lua/barbar/groups.lua

```diff
@@ -6,29 +6,30 @@ local table_insert = table.insert
 local strcharpart = vim.fn.strcharpart --- @type function
 local strwidth = vim.api.nvim_strwidth --- @type function
 
-local utils = require'barbar.utils'
+--- @class barbar.groups.item
+--- @field hl string the highlight group to use
+--- @field text string the content being rendered
```

Comment:
I'm not sold on the group/item change.

For the current "group", the names I prefer are: node (from html/dom) or atom.

For the current "group clump", the names I prefer are: container or segment.